app/client/components/MMAPage.tsx

```diff
@@ -510,8 +537,27 @@ const MMARouter = () => {
 	);
 };
 
+const getMvtId = (): number => {
+	const mvtId = getCookie('GU_mvt_id');
+
+	if (mvtId) {
+		return parseInt(mvtId);
+	} else {
+		return 0;
+	}
+};
```

Comment:
how about:

```return mvtId ? parseInt(mvtId) : 0;```

Reply:
I know some people don't like ternaries, but I think it makes sense in this case as it's nice and concise. Done ✅